student.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+require_relative 'person'
+
+class Student < Person
+  def initialize(name, age, parent_permision, classroom)
+    super(name, age, parent_permision)
+    @classroom = classroom
+  end
```

Comment:
- [ ] [REQUIRED] Good job so far 👍 I appreciate what you have done but in this `def` it will be clean if you do something like the method below too😃:

```ruby
...
  def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end
...
```

Reply:
Fixed.